src/components/Code/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
```

Comment:
to some extent we are reimplementing prism here a bit (parsing custom DSL - dvc commands, usage, etc). I wonder if we could use prism hooks instead - https://prismjs.com/extending.html#writing-plugins ?

Reply:
I think we already had them.
I did so because I found it relatively easy to parse the raw data and add dvc commands, usage, etc along with links to args then working with prismjs encoded data.

Reply:
> I think we already had them.

yep, there are different ways to hook into prism though as far as I understand. Not 100% sure though. 

This way we are reimplementing prism itself a bit.

Reply:
Found that Prism has an [autolinker plugin](https://github.com/PrismJS/prism/blob/master/plugins/autolinker/prism-autolinker.js) that shows how Prism hooks can be used to add links. I think it could be worth the extra work of converting this React-based children replacement since what's going on here really is quite similar to Prism.

Reply:
Mentioned on comment, did that extra work, and ended up using prism hooks.